src/rngs/small.rs

```diff
@@ -15,22 +15,28 @@ type Rng = ::rand_pcg::Pcg64Mcg;
 #[cfg(not(all(all(rustc_1_26, not(target_os = "emscripten")), target_pointer_width = "64")))]
 type Rng = ::rand_pcg::Pcg32;
 
-/// An RNG recommended when small state, cheap initialization and good
-/// performance are required. The PRNG algorithm in `SmallRng` is chosen to be
-/// efficient on the current platform, **without consideration for cryptography
-/// or security**. The size of its state is much smaller than for [`StdRng`].
+/// A small-state, fast non-crypto PRNG
 ///
-/// Reproducibility of output from this generator is however not required, thus
-/// future library versions may use a different internal generator with
-/// different output. Further, this generator may not be portable and can
-/// produce different output depending on the architecture. If you require
-/// reproducible output, use a named RNG.
-/// Refer to [The Book](https://rust-random.github.io/book/guide-rngs.html).
-/// 
+/// `SmallRng` may be a good choice when a PRNG with small state, cheap
+/// initialization, good statistical quality and good performance are required.
+/// It is **not** a good choice when security against prediction or
+/// reproducibility are important.
 ///
-/// The current algorithm is [`Pcg64Mcg`][rand_pcg::Pcg64Mcg] on 64-bit platforms with Rust version
-/// 1.26 and later, or [`Pcg32`][rand_pcg::Pcg32] otherwise. Both are found in
-/// the [rand_pcg] crate.
+/// This PRNG is **feature-gated**: to use, you must enable the crate feature
+/// `small_rng`.
```

Comment:
It is a bit funny to mention this in this place. You will never see this documentation unless you enable the feature in the first place. Maybe the RNG guide and the Readme are a better place to mention the feature?

Reply:
You do, because our online docs (and probably the docs.rs ones too) are built with all features enabled.